eslint.config.mjs

```
@@ -1,6 +1,8 @@
+import { FlatCompat } from "@eslint/eslintrc";
 import { dirname } from "path";
 import { fileURLToPath } from "url";
-import { FlatCompat } from "@eslint/eslintrc";
+import simpleImportSort from 'eslint-plugin-simple-import-sort';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)